app/main/controller/tag_controller.py

```diff
@@ -0,0 +1,21 @@
+# endpoint for tag related operations
+from flask_restplus import Resource
+from app.main import db
+from app.main.models.tags import Tag
+
+from ..util.tagDto import TagDto
+
+
+api = TagDto.api
+_tag = TagDto.tag
+
+# Routes
+
+@api.route('/tag/getAll')
+class AllTags(Resource):
+    @api.doc('list of all tags')
+    @api.marshal_list_with(_tag)
+    def get(self):
+        """Gets all tags"""
+        # Make db call
```

Comment:
Why have you left this? There are methods already present for the same.

Reply:
Yeah I missed #31. Will do this once it is merged